tests/wpt/metadata/MANIFEST.json

```diff
@@ -15226,6 +15226,10 @@
         "url": "/dom/nodes/prepend-on-Document.html"
       },
       {
+        "path": "dom/nodes/remove-row.html",
```

Comment:
This test should go in its proper place: `tests/wpt/web-platform-tests/html/semantics/tabular-data/the-table-element/`.